wercker.yml

```diff
@@ -48,12 +50,15 @@ lint:
        --enable=deadcode \
        --enable=gofmt \
        --enable=goimports \
-       --tests $(glide novendor)
+       --tests --deadline=50s ./...
```

Comment:
Not really sure why, but after switching to `dep` the `vet` and `vetshadow` linters take around 40 seconds to run on Wercker...